korge/src/commonMain/kotlin/com/soywiz/korge/view/Sprite.kt

```diff
@@ -119,11 +178,18 @@ open class Sprite(
         }
 
     private var currentAnimation: SpriteAnimation? = null
+    private var currentAnchorXY = FastArrayList<Anchor>()
```

Comment:
Since this is private and in a class that could be instantiated and accessed many times. What would you think about using `PointArrayList` instead? 
This one: https://github.com/korlibs/korma/blob/176dd4e62522d3f654ee21c383c8ecdf116b9074/korma/src/commonMain/kotlin/com/soywiz/korma/geom/PointArrayList.kt#L31

The point of that class is that you can store two X and Y values without allocating, and keeping the memory contiguous to take advantage of CPU caches.
By using Anchor you are allocating the array then separate objects for each value. Additionally, that forces Anchor to make its fields mutable

Reply:
Thanks for pointing me towards it. I will check how to use it here.